packages/share_plus/share_plus_platform_interface/lib/method_channel/method_channel_share.dart

```diff
@@ -115,6 +114,17 @@ class MethodChannelShare extends SharePlatform {
     return ShareResult(result, _statusFromResult(result));
   }
 
+  /// Closes the share sheet. Currently only supported on iOS.
+  @override
+  Future<void> close() {
+    if (defaultTargetPlatform != TargetPlatform.iOS) {
+      // Currently not supported outside of iOS
+      return Future<void>.value();
+    }
```

Comment:
If we go with the feature, I'd probably add an `assert` here, so the method fails in debug mode.